pyproject.toml

```diff
@@ -71,16 +71,23 @@ strict = true
 lines_between_types = 1
 lines_after_imports = 2
 multi_line_output = 5
-known_first_party = "mesonpy"
+known_first_party = 'mesonpy'
```

Comment:
Can you please stick to the existing style for string quotes in an existing file? Having the quoting switch forth and back between the allowed styles makes the diff harder to parse.

Reply:
The existing style was single quotes. I didn't realize that, so in #211 I used double quotes (probably because of this line - the only one in the file that was double quotes, but it was at the bottom where I was working). I was just trying to get the file back into a consistent format (even if it's the opposite of what I usually do, I usually follow black).

Eventually there should be a good autoformatter and this will never come up.

Reply:
I removed this one line's change, but kept the pytest normalization to single quotes - let me know if you'd like me to leave that double, too.

Reply:
Sorry, I didn't realize. Please segregate coding style adjustments to their own commits. Among other things, this avoid this kind misunderstandings to occur.

Reply:
Split this up.